google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -42,42 +43,64 @@ public StorageClass apply(String constant) {
       new StringEnumType(StorageClass.class, CONSTRUCTOR);
 
   /**
-   * Standard storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   * Standard storage class.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/storage/docs/storage-classes#standard">https://cloud.google.com/storage/docs/storage-classes#standard</a>
    */
   public static final StorageClass STANDARD = type.createAndRegister("STANDARD");
 
   /**
-   * Nearline storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   * Nearline storage class.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/storage/docs/storage-classes#nearline">https://cloud.google.com/storage/docs/storage-classes#nearline</a>
    */
   public static final StorageClass NEARLINE = type.createAndRegister("NEARLINE");
 
   /**
-   * Coldline storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   * Coldline storage class.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/storage/docs/storage-classes#coldline">https://cloud.google.com/storage/docs/storage-classes#coldline</a>
    */
   public static final StorageClass COLDLINE = type.createAndRegister("COLDLINE");
 
   /**
-   * Archive storage class. See: https://cloud.google.com/storage/docs/storage-classes for details
+   * Archive storage class.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/storage/docs/storage-classes#archive">https://cloud.google.com/storage/docs/storage-classes#archive</a>
    */
   public static final StorageClass ARCHIVE = type.createAndRegister("ARCHIVE");
 
   /**
    * Regional storage class. This is supported as a legacy storage class and will be deprecated in
-   * the future. See: https://cloud.google.com/storage/docs/storage-classes for details
+   * the future. This class cannot be set. Unless you already are using it, you should use {@link
+   * #STANDARD} instead.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/storage/docs/storage-classes#legacy">https://cloud.google.com/storage/docs/storage-classes#legacy</a>
    */
   public static final StorageClass REGIONAL = type.createAndRegister("REGIONAL");
 
   /**
    * Multi-regional storage class. This is supported as a legacy storage class and will be
-   * deprecated in the future. See: https://cloud.google.com/storage/docs/storage-classes for
-   * details
+   * deprecated in the future. This class cannot be set. Unless you already are using it, you should
```

Comment:
Legacy storage classes can still be set, please remove `This class cannot be set` for all legacy storage classes.

Additionally, the issue that didn't across in my issue is that a developer at first only see's `Mult-regional storage class.` and not the rest. 

![ySdN1VJtAXu](https://user-images.githubusercontent.com/1907648/79599826-03303a80-809b-11ea-8c6f-fa3245cff6ca.png)

The additional information isn't available until scrolling down:

![FAuvGNOgqhx](https://user-images.githubusercontent.com/1907648/79599879-16dba100-809b-11ea-9140-d42a9749a96a.png)

If it's being truncated because of the period `.`. then I recommend this:

> Legacy Multi-regional storage class, use REGIONAL storage class for new buckets.

Reply:
> Legacy Multi-regional storage class, use REGIONAL storage class for new buckets.

1) Do you really mean REGIONAL?

https://cloud.google.com/storage/docs/storage-classes#legacy recommends STANDARD class:
> Unless you already are using one of these additional classes, you should use Standard Storage instead.

2) A storage class can be set not only for buckets, but for object as well. And not only for new, but for existing. Right? May be it's not correct to say `for new buckets`

Reply:
:man_facepalming: I meant STANDARD. 

Good catch (blobs vs buckets), then use:

> Legacy {Storage class} storage class, use STANDARD storage class instead.

Reply:
![legacyStorageClasses1](https://user-images.githubusercontent.com/58846611/79602786-d0666080-80f3-11ea-8745-5f9b5f536790.png)

...

![legacyStorageClasses2](https://user-images.githubusercontent.com/58846611/79602795-d5c3ab00-80f3-11ea-8790-4c53da654caf.png)